bucket/figma.json

```diff
@@ -0,0 +1,25 @@
+{
+    "version": "101.6",
+    "description": "Figma connects everyone in the design process so teams can deliver better products, faster.",
+    "homepage": "https://www.figma.com/",
+    "license": {
+        "identifier": "Shareware",
+        "url": "https://www.figma.com/summary-of-policy/"
+    },
+    "url": "https://desktop.figma.com/win/FigmaSetup.exe",
+    "hash": "e3b0c44298fc1c149afbf4c8996fb92427ae41e4649b934ca495991b7852b855",
+    "bin": "FigmaSetup.exe",
```

Comment:
What does FigmaSetup exe do?

If its a setup, shouldn't it already be extracted?

Could you share screenshots?

Reply:
- It installs the designing tool into system.
- Yup, it's an executable not an archive (afaik they don't provide one).

![image](https://user-images.githubusercontent.com/38415384/139584305-772a41e2-300f-4671-be00-7ae6a34a00a3.png)

Reply:
Could you run Scoop Install and then open the shortcut and show us what happens?

Scoop supports extraction from a lot of EXE based installers as well

Reply:
![image](https://user-images.githubusercontent.com/38415384/139624285-63563246-2ae5-48f1-af93-70da622c7cc3.png)

is this what you're looking for?